lib/cocaine.rb

```diff
@@ -1,12 +1,12 @@
 # coding: UTF-8
 
-require 'rbconfig'
-require 'cocaine/os_detector'
-require 'cocaine/command_line'
-require 'cocaine/command_line/output'
-require 'cocaine/command_line/multi_pipe'
-require 'cocaine/command_line/runners'
-require 'cocaine/exceptions'
+require 'terrapin'
```

Comment:
arent double quotes needed when you use interpolation so using doubled is 0.0000001 % slower since ruby checks if there is interpolation in doubled quotes?